src/common/checksum.c

```diff
@@ -222,10 +254,11 @@ daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
 
 int
 daos_csummer_type_init(struct daos_csummer **obj, enum DAOS_CSUM_TYPE type,
-		       size_t chunk_bytes, bool srv_verify)
+		       size_t chunk_bytes, bool srv_verify, bool dedup,
+		       bool dedup_verify, size_t dedup_bytes)
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?